esp-hal/src/dma/pdma.rs

```diff
@@ -201,6 +206,10 @@ impl<C: PdmaChannel<RegisterBlock = SpiRegisterBlock>> RegisterAccess for SpiDma
             .modify(|_, w| unsafe { w.inlink_addr().bits(address) });
     }
 
+    fn set_auto_wrback(&self, _enable: bool) {
```

Comment:
I've noticed you assert in the TX one and no-op here.

I'm wondering if `set_auto_wrback` should be moved down from `RegisterAccess` to `TxRegisterAccess`, since it doesn't seem to apply to RX. (Unless the P4 and C5 do something different, no public TRM so I can't check)

Reply:
+1 - since for reads the descriptor needs to be written anyway to set length it's always done (haven't checked C5 but I see no reason they won't do it)